app/src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { useAuth } from "./AppProviders";
-import AuthenticatedApp from "./components/AuthenticatedApp";
-import UnauthenticatedApp from "./components/UnauthenticatedApp";
+import AuthenticatedApp from "./components/Authentication/AuthenticatedApp";
+import UnauthenticatedApp from "./components/Authentication/UnauthenticatedApp";
```

Comment:
These aren't authentication really, they're the entire app

Reply:
Sure, should we have a folder called `App Containers` and add these two in there? @thisisjofrank

Reply:
That sounds good :)